jobs/openshift/cluster/create/openshift-cluster-create.yaml

```diff
@@ -35,15 +35,15 @@
           description: '[REQUIRED] the AWS instance type for infra node(s)'
       - string:
           name: 'master_group_size'
-          default: '3'
+          default: '1'
           description: '[REQUIRED] The amount of AWS master nodes to be created'
       - string:
           name: 'compute_group_size'
-          default: '3'
+          default: '2'
           description: '[REQUIRED] The amount of AWS compute nodes to be created'
       - string:
           name: 'infra_group_size'
-          default: '3'
+          default: '2'
```

Comment:
I'd leave these values as they were for the time being, we have a small issue to resolve before we can change these https://issues.jboss.org/browse/INTLY-1675. We also probably need the compute size to be 3.